MS Teams Integration (#3467)
owncloud/docs: modules/user_manual/pages/integration/ms-teams.adoc

= ownCloud Integration into Microsoft Teams
:toc: right

== Introduction

You can access your ownCloud via Microsoft Teams, if your administrator has created an app available in your organization's app catalog. It is possible, that the admin already has pre-enabled the app for the users, in this case you do not need to search for it in your organization's app catalog as it is already pinned.

== Access the App

> To access the app, use following steps:

Reviewer: ```suggestion
To access the app, perform the following steps:
```

. Log in to Teams.

. Click on the overflow menu, select Apps and search for *ownCloud for Teams* or the name provided by your admin.
+
image:integration/ms-teams/owncloud-teams-app.png[,width=60%]
+
TIP: You can pin the app to your left sidebar permanently. Right-click on the icon and select "Pin".

. Log in to ownCloud by clicking on *Login to ownCloud with Azure AD*.
+
image:integration/ms-teams/login-to-owncloud-with-azure-ad.png[,width=80%]

. For security reasons, you need to repeat the login process after a certain idle time.

> . Repeat the steps if there are more than one ownCloud for Teams apps available accessing different ownCloud hosts

Reviewer: ```suggestion
. Repeat the steps if there is more than one ownCloud for Teams app available accessing different ownCloud hosts.
```
